src/lwt_support.ml

```diff
@@ -23,21 +23,22 @@ let (>>=) = Lwt.bind
 let really_write fd str =
   let len = String.length str in
   let rec inner written =
-    Lwt_unix.write fd str written (len-written) >>= (fun n ->
+    Lwt_unix.write fd (Bytes.unsafe_of_string str) written (len-written)
```

Comment:
This `unsafe_of_string` is also safe as `Lwt_unix.write` should not mutate the given byte sequence but treat it as read-only.